web_sheet_full_width/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Form view sheet at full width
+=============================
+
+This module was written to extend the functionality of the web client
+to get full width in the form view sheet.
+
+This module works in community edition and in enterprise edition.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
```

Comment:
Wrong version

Reply:
That of web_responsive is also wrong

Reply:
Indeed, changed in https://github.com/OCA/web/commit/ebdbd58e2b7ff0dc6285e4a0fa301387fa674609